### ISSUE TITLE
Help for sub parsers isn't shown.

### ISSUE THREAD

**Issue by richburdon** [nodeca/argparse#115]
After adding subparsers with addHelp, no help is shown below the top-level.

**Comment by puzrin** [comment]
1. How should it work (in python)?
2. Please, provide code sample how to reproduce

**Comment by richburdon** [comment]
Example:

~~~~
    let db = commands.addParser('db', { addHelp: true });
    let dbCommand = db.addSubparsers({
      title: 'database',
      dest: 'db_command'
    });

    dbCommand.addParser('users',    { addHelp: true });
    dbCommand.addParser('groups',   { addHelp: true });
~~~~

Output:

~~~~
commands:
  {db}
~~~~

Subcommands of db ("users", "groups") not shown.

**Comment by M3TIOR** [comment]
This is how it should look via Python. I noticed you ask that question often @puzrin . If you're the maintainer you should attempt / ask your manager to have an issue template added to the Github Issues submission page which will automatically ask people that question so you don't have to later. 
![Screenshot_2020-04-18_00-18-53](https://user-images.githubusercontent.com/5925208/79630934-5b525580-810a-11ea-8d44-d9ad363a584c.png)

**Comment by M3TIOR** [comment]
@nodeca

**Comment by rlidwka** [comment]
We've ported everything from scratch based on python 3.9 (released as argparse version 2.0).

I'm pretty sure it's fixed there.

---

as a side note:

> to have an issue template added to the Github Issues submission page which will automatically ask people that question so you don't have to later

I honestly believe that a few seconds it takes to ask a question (*when necessary*) is far better than give everyone a clear impression of Vogon-style bureaucracy in action.
